docs/50_deploy.md

```diff
@@ -13,3 +13,7 @@ To make the command work, also the following flags described in [options](./opti
 
 - `--namespace`: to specify the namespace in which the resources are deployed
 - The set of flags required to connect to the Kubernetes cluster
+
+## Await jobs completion
+
+During the deploy phase resources are applied sequentially one after the other. `mlp` allows to wait for the completion of a `Job` using the annotation `mia-platform.eu/await-completion` on the specific resource to be awaited. The annotation value is ignored and therefore can be set to any value, we recommend using an empty string. Job resources using this annotation will be applied by `mlp` then the tool will listen to cluster events until the resource status becomes `Completed`. The waiting phase has a timeout of `30s`. If the resource status is not completed after that waiting period an error is fired up and the process exits.
```

Comment:
Why an empty string? It seems like an error in the configuration.
Given that we have a timeout, how do you think about set as value the await timeout as value?

Reply:
LGTM :+1: